OSPSuite.Presentation.Importer/Views/ColumnMappingControl.cs

```diff
@@ -202,7 +202,7 @@ private void fillComboBoxItems(ImageComboBoxEdit editor, DataFormatParameter map
          //MetaData
          foreach (var category in _metaDataCategories)
          {
-            if (!_mappings.Any(m => m.Configuration.Type == ParameterConfiguration.DataFormatParameterType.MetaData && m.ColumnName == category.DisplayName))
+            if (!_mappings.Any(m => m.Type == ParameterConfiguration.DataFormatParameterType.MetaData && m.ColumnName == category.DisplayName))
```

Comment:
That's a lot of code that does not belong into the view. THis needs to be deals with at the presenter level IMO